src/cdk-experimental/menu/menu-item-interface.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {EventEmitter} from '@angular/core';
+
+/**
```

Comment:
I would add a comment in this file that explains that it exists in order to avoid circular dependencies